x/oracle/keeper/migrations.go

```diff
@@ -22,3 +23,12 @@ func (m Migrator) Migrate1to2(ctx sdk.Context) error {
 	m.keeper.SetMaximumMedianStamps(ctx, 1)
 	return nil
 }
+
+// Migrate2to3 migrates from version 2 to 3.
+func (m Migrator) Migrate2to3(ctx sdk.Context) error {
+	m.keeper.SetHistoricStampPeriod(ctx, types.DefaultHistoricStampPeriod)
```

Comment:
We should probably update these default values (I set them a while ago before a lot of discussions around what they should be around). Like do we want to start with the `HistoricStampPeriod` being equal to the `VotePeriod` for example? Right now the `DefaultHistoricStampPeriod` is a lot bigger than that.

Reply:
Yeah let's decide that in this PR - I think that might be reasonable. 24 hour historic stamps, 3 hour median?
Might want to talk about this in slack @rbajollari

Reply:
Added in 4c99f1cc3c5ecd6cce3a2ef2eb27b28aacbf2b9c

Reply:
UP, let's address: https://github.com/umee-network/umee/pull/1656#pullrequestreview-1218175261